index.css

```diff
@@ -0,0 +1,867 @@
+  top: 40px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 24px;
+
+  text-align: center;
+  letter-spacing: 0.1px;
+
+  color: #252b42;
+}
+
+.contents__description {
+  width: 455px;
+  height: 57px;
+  left: 0px;
+  top: 64px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 57px;
+
+  letter-spacing: 0.2px;
+  color: #252b42;
+}
+
+@media (max-width: 1100px) {
+  .appointment {
+    position: relative;
+  }
+}
+
+.input {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
+  gap: 10px;
+}
+.input__item {
+  box-sizing: border-box;
+  left: 0px;
+  right: 0px;
+  top: 0px;
+  bottom: 0px;
+  padding: 10px;
+  background: #f9f9f9;
+  border: 1px solid #bdbdbd;
+  border-radius: 5px;
+}
+
+.contents__area {
+  box-sizing: border-box;
+  width: 598px;
+  height: 140px;
+  background: #f9f9f9;
+  border: 1px solid #bdbdbd;
+  border-radius: 5px;
+}
+
+.contents__btn {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 10px 36px;
+  gap: 10px;
+  width: 185px;
+  height: 48px;
+  border-radius: 2px;
+
+  background: #ffa62b;
+  color: white;
+  border: none;
+}
+
+.last__btn {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 10px 36px;
+  gap: 10px;
+  width: 185px;
+  height: 48px;
+  border-radius: 2px;
+
+  background: #ffa62b;
+  color: white;
+  border: none;
+}
+
+.last {
+  padding-top: 30px;
+  padding-left: 120px;
+  padding-right: 120px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.last__title {
+  width: 454px;
+  height: 32px;
+
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+
+  letter-spacing: 0.1px;
+
+  color: #252b42;
+}
+.last__description {
+  width: 273px;
+  height: 20px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 20px;
+  letter-spacing: 0.2px;
+  color: #737373;
+}
+
+.footer-container {
+  background-color: #252b42;
+  height: fit-content;
+}
+
+.footer {
+  width: 70%;
+  margin: 0 auto;
+  margin-top: 30px;
+}
+
+.footer__header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 3rem 0;
+  border-bottom: 1px solid #737373;
+}
+
+.footer__title {
+  color: white;
+}
+
+.footer__icons-container {
+  display: flex;
+  gap: 1rem;
+  align-items: center;
+}
+
+.footer__menu {
+  display: flex;
+  justify-content: space-between;
+  gap: 1rem;
+  padding: 5rem 0;
+}
+
+.footer__card-title {
+  color: white;
+  font-size: 19px;
+}
+
+.footer__card-menu {
+  color: #bdbdbd;
+  list-style-type: none;
+  font-size: 15px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 1rem;
+  padding-left: 0;
+}
+
+.footer__card-input {
+  outline: none;
+  padding: 0 10px;
+  width: 10rem;
+}
+
+.footer__card-form {
+  display: flex;
+  gap: 0;
+}
+
+.footer__card-btn {
+  border-top-left-radius: none !important;
```

Comment:
Avoid using `!important` to increase maintainability of your code. Try to increase the specificity with selectors not the `!important`